contracts/test/EVMPrecompileTester.js

```diff
@@ -173,6 +173,45 @@ describe("EVM Test", function () {
                 // TODO: Add staking query precompile here
             });
         });
+
+        describe("EVM Oracle Precompile Tester", function () {
+            const OraclePrecompileContract = '0x0000000000000000000000000000000000001008';
+            before(async function() {
+                const exchangeRatesContent = readDeploymentOutput('oracle_exchange_rates.json');
+                const twapsContent = readDeploymentOutput('oracle_twaps.json');
+
+                exchangeRatesJSON = JSON.parse(exchangeRatesContent).denom_oracle_exchange_rate_pairs;
+                twapsJSON = JSON.parse(twapsContent).oracle_twaps;
+
+                const [signer, _] = await ethers.getSigners();
+                owner = await signer.getAddress();
+
+                const contractABIPath = path.join(__dirname, '../../precompiles/oracle/abi.json');
+                const contractABI = require(contractABIPath);
+                // Get a contract instance
+                oracle = new ethers.Contract(OraclePrecompileContract, contractABI, signer);
+            });
+
+            it("Oracle Exchange Rates", async function () {
+                const exchangeRates = await oracle.getExchangeRates();
```

Comment:
NOTE: For both oracle exchange rates and twaps will compare the exact values, but even if querying at a specific height its complicated with hardhat. May have to create another contract that emits events or a wrapper where we can specify a height